kratos/python_scripts/vtk_output_process.py

```diff
@@ -8,9 +8,9 @@ def Factory(settings, model):
     return VtkOutputProcess(model, settings["Parameters"])
 
 
-class VtkOutputProcess(KratosMultiphysics.Process):
+class VtkOutputProcess(KratosMultiphysics.OutputProcess):
     def __init__(self, model, settings):
-        KratosMultiphysics.Process.__init__(self)
+        super().__init__(self)
```

Comment:
```suggestion
        super().__init__()
```
Apparently is failing, the pybind documentation says this is complicated